src/dehydrated_devices.rs

```diff
@@ -21,6 +21,51 @@ impl From<dehydrated_devices::DehydratedDevices> for DehydratedDevices {
     }
 }
 
+/// Dehydrated device key
+#[wasm_bindgen]
+#[derive(Debug)]
+pub struct DehydratedDeviceKey {
+    pub(crate) inner: Uint8Array,
+}
+
+#[wasm_bindgen]
+impl DehydratedDeviceKey {
+    /// Generates a new random pickle key.
+    #[wasm_bindgen(js_name = "createRandomKey")]
+    pub fn create_random_key() -> Result<DehydratedDeviceKey, JsError> {
+        Ok(SdkDehydratedDeviceKey::new()?.into())
+    }
+
+    /// Generates a new random pickle key.
```

Comment:
if we're passing in the array, it's not random?

Reply:
sry. bsd copy paste